styles.css

```diff
@@ -112,3 +112,532 @@ h1 {
   padding-bottom: 0;
   padding-left: 16px;
 }
+
+.rectangle21 {
+  width: 320px;
+  height: 186px;
+  left: 0;
+  top: 754px;
+}
+
+.works {
+  display: grid;
+  grid-template-columns: auto auto auto;
+  grid-template-rows: auto auto auto;
+  background-color: #1a2236;
+  width: 320px;
+  height: 156px;
+  left: 0;
+  top: 940px;
+  text-align: center;
+  padding-top: 10px;
+  padding-left: 0;
+  justify-content: center;
+  column-gap: 10px;
+  row-gap: 15px;
+}
```

Comment:
- [ ] Having made a great effort working on the work section of your project, Currently, I noticed your design is kind of different from what we have in the Figma template. Below is the highlight of where your attention is needed most.

- Kindly make the `button`, `technology`, and the `project name` aligned together as we have it in the Figma design
- Kindly remove the underline in the `see project` button, also the `font size`, and `width` of the button so that it looks like what we have in the Figma design
- Kindly check the `font family`, `font size`, and `font weight` of the technology and make sure they look exactly like what we have in the Figma design.

Your design 👇 

![mel1](https://user-images.githubusercontent.com/78634841/213314169-835d8856-48e2-410a-9f30-9759074e6cd4.png)

Figma design 👇 

![temp5-real](https://user-images.githubusercontent.com/78634841/213314245-f953be95-21e4-4727-bbc9-148a6e3beb56.png)